src/core/sys/posix/sys/types.d

```diff
@@ -1404,6 +1404,11 @@ else
     static assert(false, "Unsupported platform");
 }
 
+version (Linux)
+{
+    alias lwpid_t = pid_t;
```

Comment:
Type definition goes in sys/procfs.d, and `linux` is not capitalized.

Reply:
There is no file named `sys/procfs.d`. Should I create one myself for the sole purpose of adding `lwpid_t` to it? Sounds a bit overkill.

Reply:
Yes, that way we have a one to one mapping of system headers to d modules. Overkill is the price we pay for consistency.